e2e/test-extension/extension.neon

```diff
@@ -0,0 +1,2 @@
+parameters:
+    invalidConfig: []
```

Comment:
using an invalid configuration key to crash phpstan if the extension was installed